POS/Operation/02_SelectOffer.md

```diff
@@ -87,34 +155,116 @@ Offers for POS are created, see [Manage offers for POS](../Integration/07_Manage
 
 #### Prerequisites
 
-Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
+- Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
+- An age restriction for the offer is defined, for example 18 or 16.    
+- The *Display age rating* setting is activated. For detailed information, see [Display age rating](../UserInterface/02a_GlobalSettings.md#display-age-rating).
 
 #### Procedure
 
-*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT*
+*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT* 
 
 ![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Cashpoint.png "[Cashpoint]")
 
-1. Activate the *Single scan* toggle in the bill list on the right side of the pay desk interface.   
+1. Select an offer.   
+    If the offer requires an age verification, the *Age verification* window is displayed.
 
-    > [Info] The single scan works from the moment it is toggled on. It does not apply backwards to products that were already on the bill list at the time of activation.   
-    Even if the single scan function is active, you can edit the quantity of products manually in the bill list, see [Edit the quantity of a product](./03_EditBillList.md#edit-the-quantity-of-a-product).
+    ![Age verification](../../Assets/Screenshots/POS/Sales/Cashpoint/AgeVerification.png "[Age verification]")
+
+2. Check the age of the customer. The *Age verification* window provides you with both the minimum age required and the date of birth that the customer must meet or fall below. Eventually, an explanatory text has been added.
+
+    ![Age verification](../../Assets/Screenshots/POS/Sales/Cashpoint/AgeVerification.png "[Age verification]")
+    
+
+3. Click the [AGE OF BUYER CONFIRMED] button if the customer is allowed to buy this offer.  
+    The offer is added to the bill list.
+
+4. Click the [CANCEL] button if the customer is too young to buy this offer.   
+    The offer is not added to the bill list.
+
+
+## Add a product option
+
+There are products where the buyer has the option of customizing them. If a store has the appropriate technical capabilities, this could be shirts with optional flocking, for example, or photos that a customer can have printed on mugs or other items. In these cases, a specific service is associated to an offer. This service is optional for customers and can be purchased additionally.
+
+#### Prerequisites
+
+- Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
+- Offers relevant to product options have been associated to the offers to which they are related, see [Create a product option](../Integration/07_ManageOffers.md#create-a-product-option).
```

Comment:
**Offers** relevant to product options have been **associated to the offers** **to which they are related** -> verstehe nicht, was gemeint ist, sorry

Reply:
done